dockerfile_scripts/torch_tb_profiler_patch.patch

```diff
@@ -13,4 +13,9 @@ index fa15705..e8a2420 100644
 +
                  except Exception as ex:
                      logger.warning("Failed to scan runs. Exception=%s", ex, exc_info=True)
++                    if not self._is_active:
```

Comment:
perhaps a `finally: ` clause will be more appropriate in this try, whatever it's doing?

Reply:
done

Reply:
I believe the upper if part can be removed then?

Reply:
not quite. there's no exception in that case. it just runs through the for loop, doesn't find anything, and will try again until the end of time.